launch/pacmod_game_control.launch

```diff
@@ -74,6 +74,7 @@
       <param name="coalesce_interval" type="double" value="0.02"/>
       <param name="default_trig_val" value="true"/>
       <param name="dev" value="/dev/input/js0"/>
+      <param name="deadzone" value="0.0"/>
     </node>
 
```

Comment:
Please change this file back to the defaults that were set. People expect these when they pull from master and I don't want to cause problems for customers.